compiler/resolution/wrappers.cpp

```diff
@@ -2113,12 +2123,28 @@ PromotionInfo::PromotionInfo(FnSymbol* fn,
 {
   int numActuals = actualFormals.size();
 
+  // Operators may mismatch the number of formals and the number of actuals.
+  // Store any difference for later.
+  if (numActuals != fn->numFormals() && fn->hasFlag(FLAG_OPERATOR)) {
+    formalToActualOpMod = fn->numFormals() - numActuals;
+  } else {
+    formalToActualOpMod = 0;
+  }
```

Comment:
it's OK to do it this way but it would make more sense to me to check that the mismatched args are what we expect (method token and then this).